CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -1,7 +1,30 @@
-# Release notes for v3.6.0
+# Release notes for v3.6.1
 
 [Documentation](https://kubernetes-csi.github.io)
 
+## Changes by Kind
+
+### Bug or Regression
+
+- CVE fixes: CVE-2023-44487, CVE-2023-39323, CVE-2023-3978 (#1057, @dannawang0221)
```

Comment:
Can you run release-notes with the "--markdown-links" flag? This way it will provide a markdown link for the PR and github id.

Reply:
done